src/test/java/uk/gov/mca/beacons/api/controllers/RegistrationsControllerIntegrationTest.java

```diff
@@ -70,13 +93,36 @@ void givenInvalidRegistration_whenPosted_thenStatus400(
       .exchange();
   }
 
-  private Map<RegistrationUseCase, Object> getRegistrationsJson()
-    throws IOException {
-    final String json = new String(
-      Files.readAllBytes(Paths.get(REGISTRATION_JSON_RESOURCE))
-    );
+  private Map<RegistrationUseCase, Object> toJson(String registrationFixture)
+    throws JsonProcessingException {
     final TypeReference<EnumMap<RegistrationUseCase, Object>> typeReference = new TypeReference<>() {};
-    return OBJECT_MAPPER.readValue(json, typeReference);
+    return OBJECT_MAPPER.readValue(registrationFixture, typeReference);
+  }
+
+  private UUID createTestAccountHolder() {
```

Comment:
do we enforce ref integrity in the db for account holder id on a beacon? If not could we just put any junk uuid in the field rather than generating one each time?